packages/edit-post/src/components/preview-options/style.scss

```diff
@@ -0,0 +1,85 @@
+.editor-post-preview__dropdown {
+	display: none;
+	margin-right: $grid-size;
+	border: 1px solid $light-gray-600;
+}
+
+.editor-post-preview__button-toggle.editor-post-preview__button-toggle.editor-post-preview__button-toggle.editor-post-preview__button-toggle.editor-post-preview__button-toggle.editor-post-preview__button-toggle.editor-post-preview__button-toggle {
+	border-radius: 0;
+	display: flex;
+	justify-content: space-between;
+	height: $block-controls-height - 2; // Needs to be 2px smaller to account for the box-shadow.
+
+	&:hover,
+	&:focus {
+		box-shadow: 0 0 0 1px $dark-gray-500, 0 0 0 2px $white;
+	}
+
+}
+
+.editor-post-preview__button-resize.editor-post-preview__button-resize {
+	padding-left: 36px;
+
+	&.has-icon {
+		padding-left: $grid-size;
+	}
+}
+
+.editor-post-preview__button-separator {
+	border-left: 1px solid $light-gray-600;
+	height: $icon-button-size;
+	padding: $grid-size 0 $grid-size $grid-size;
+	margin-left: $grid-size;
+}
+
+.editor-post-preview__dropdown-content {
+	.components-popover__content {
+		overflow-y: visible;
+	}
+	.components-menu-group__label {
+		padding-left: 15px;
+	}
+	.components-menu-group + .components-menu-group {
+		border-top: 1px solid $light-gray-500;
+		padding: 0;
+	}
+}
+
+.editor-post-preview__grouping-external {
+	position: relative;
+}
+
+.editor-post-preview__button-external.editor-post-preview__button-external.editor-post-preview__button-external.editor-post-preview__button-external.editor-post-preview__button-external.editor-post-preview__button-external.editor-post-preview__button-external {
```

Comment:
As in https://github.com/WordPress/gutenberg/pull/19082/files#r377573579

Reply:
Same as above!